src/myMethods.js

```diff
@@ -0,0 +1,58 @@
+const prepareData = arr => {
+  return arr.map(person => {
+    const sortedPerson = {};
```

Comment:
```suggestion
    const sortedPerson = {
      ...person,
      id: ..
      age: ...
      otherCustomField: ...
    };
```
You can copy all fields from previous object via spread operator